src/preact/test-utils.js

```diff
@@ -0,0 +1,53 @@
+import * as preact from /*OK*/ 'preact';
```

Comment:
I'm not sure this is the correct place for this file to live, this seems like something that should go under `testing`

Reply:
Agreed.

Reply:
Good thinking! Moved to `testing/preact`.